app/api/helpers/permissions.py

```diff
@@ -0,0 +1,91 @@
+from functools import wraps
+from flask import current_app as app
+from flask_jwt import _jwt_required, current_identity
+from app.api.helpers.errors import ForbiddenError
+
+
+def decdec(inner_dec):
```

Comment:
I am not very sure but the variable names seem very confusing. It should be more intuitive I think.